demos/_unit-test/callback_2.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+declare(strict_types=1);
+/**
+ * Behat testing.
+ * Test for callback in callback.
+ */
+
+namespace atk4\ui\demo;
+
+use atk4\ui\Button;
+use atk4\ui\Crud;
+use atk4\ui\Header;
+use atk4\ui\Loader;
+
+/** @var \atk4\ui\App $app */
+require_once __DIR__ . '/../init-app.php';
+
+$m = (new CountryLock($app->db))->setLimit(5);
+$m->getUserAction('edit')->ui['button'] = new Button(['Edit', ['ui' => 'atk-test button']]);
+
+$loader = Loader::addTo($app);
+$loader->loadEvent = false;
+
+$loader->set(function($p) use ($m) {
```

Comment:
should be CRUD (as it uses Popup) and probably it should be defined with function, which can be called unlimited times

Reply:
This test is failing using the current develop branch. So should be enough for now.
But feel free to supply your own sample.

Reply:
basically in `reload_not_working_after_upload__add_submit_issue` branch, `addCrudDemo`, can be very simple, I even do not think we need upload, but it will be nice if crud (like there) can be used for input for field, this is quite complex test which will be needed if we will drop sticky global later...

the field should be used twice and both fields should be edited twice (to simulate issues with with sticky...)